docker-compose/database/db-changes/datasets/0030_add_subjectType_column.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE subject_study ADD COLUMN subject_type int(11) DEFAULT NULL;
```

Comment:
Nothing is done to initialize this column (and the equipment name too)
Maybe you should add a "one shot" script to do so :) -> You can do cross DB sql request here